blockonomics-woocommerce.php

```diff
@@ -155,26 +162,28 @@ function add_page()
             'Blockonomics', 'Blockonomics', 'manage_options',
             'blockonomics_options', 'show_options'
         );
-
-        if (get_option('blockonomics_api_updated') == 'true' && isset($_GET['settings-updated']) ? $_GET['settings-updated'] : '' == 'true')
+        
+        if (get_option('blockonomics_redirect') == "yes")
```

Comment:
The `blockonomics_redirect` is as saved as part of the form so that we know if we should redirect the user after submitting the form. 

If a user has made changes to the form and then clicks a different tab within the plugin the form will be submitted and `blockonomics_redirect` will be saved as 'yes'. After the form is submitted a check will be done and if the user should be redirected `blockonomics_redirect` will be set back to 'no' and the redirect take place.